### PR TITLE
[NMS] Add subscriber traffic policy config

### LINKED ISSUE #2836: Edit subscriber traffic policy config
https://www.figma.com/proto/aRJzvSa3J9nhD2n3yERd1s/Magma---Phase-2-(Updated-Styles)?node-id=244%3A1152&scaling=min-zoom

### PR DESCRIPTION
**Pull request by HannaFar**
Signed-off-by: HannaFar <hannafarag159@gmail.com>

## Summary

Add subscriber traffic policy config edit tab.
Move APN static IP tab in subscriber config edit.
Add APNs static IP in subscriber detail page.

Capture d’écran 2020-11-17 à 16 53 35

Capture d’écran 2020-11-17 à 16 55 20
Capture d’écran 2020-11-17 à 16 55 30
Capture d’écran 2020-11-17 à 16 55 41

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: nms/app/packages/magmalte/app/views/subscriber/SubscriberDetailConfig.js [hunk 1]
```diff
@@ -122,24 +122,82 @@ export default function SubscriberDetailConfig() {
...
+      <SubscriberEditDialog
+        open={open}
+        onClose={() => setOpen(false)}
+        editProps={{editTable: 'staticIps'}}
+      />
+      <Button component="button" variant="text" onClick={() => setOpen(true)}>
+        {'Edit'}
+      </Button>
+    </>
+  );
```
Comment (andreilee) [inline nms/app/packages/magmalte/app/views/subscriber/SubscriberDetailConfig.js]: I think that here it's better to increase code re-use by combining these three into a single component with an enum prop field to control which tab the dialog opens up with

File: nms/app/packages/magmalte/app/views/subscriber/SubscriberAddDialog.js [hunk 2]
```diff
@@ -550,305 +792,73 @@ type subscriberStaticIpsRowType = {
...
+                ...props.subscriberState.lte,
+                sub_profile: value,
+              });
+            }}
+          />
+        </AltFormField>
+        <AltFormField label={'Auth Key'}>
+          <PasswordInput
+            className={classes.input}
+            placeholder="Enter Auth Key"
```
Comment (andreilee) [inline nms/app/packages/magmalte/app/views/subscriber/SubscriberAddDialog.js]: Have the placeholder show the input format

File: nms/app/packages/magmalte/app/views/subscriber/SubscriberAddDialog.js [hunk 3]
```diff
@@ -550,305 +792,73 @@ type subscriberStaticIpsRowType = {
...
+            value={props.authKey}
+            error={props.authKey && !isValidHex(props.authKey) ? true : false}
+            onChange={v => props.setAuthKey(v)}
+          />
+        </AltFormField>
+        <AltFormField label={'Auth OPC'}>
+          <PasswordInput
+            value={props.authOpc}
+            placeholder="Enter Auth OPC"
```
Comment (andreilee) [inline nms/app/packages/magmalte/app/views/subscriber/SubscriberAddDialog.js]: Have the placeholder show the input format

File: nms/app/packages/magmalte/app/views/subscriber/SubscriberApnStaticIpsEdit.js [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+                      <ListItemText primary={apn} />
+                    </MenuItem>
+                  ))}
+                </Select>
+              </FormControl>
+            </AltFormField>
+            <AltFormField label={'APN Static IP'}>
+              <OutlinedInput
+                className={classes.input}
+                placeholder="Enter IP"
```
Comment (andreilee) [inline nms/app/packages/magmalte/app/views/subscriber/SubscriberApnStaticIpsEdit.js]: Show input format of IPv4 and example

**Review by andreilee — CHANGES_REQUESTED** [review]
Set the height of the content in the three dialog tabs to the same value, just as a minor usability improvement

**Review by karthiksubraveti — COMMENTED** [review]
Sorry for the late review here. Here instead of showing there is 1 active IP, i wonder if it might be better to show a table with a apn and the actual IP address which was configured. That might be better.

**Review by andreilee — APPROVED** [review]

**Pull request merged**